.github/workflows/haskell-ci.yml

```diff
@@ -203,6 +204,8 @@ jobs:
           $HCPKG list --simple-output --names-only | perl -ne 'for (split /\s+/) { print "constraints: any.$_ installed\n" unless /^(swarm)$/; }' >> cabal.project.local
           cat cabal.project
           cat cabal.project.local
+          echo "::set-output name=SWARM_PKGDIR::${PKGDIR_swarm}"
```

Comment:
As @nitinprakash96 [pointed out to me](https://github.com/swarm-game/swarm/issues/1970#issuecomment-2226885910), `set-output` is deprecated.